articles/quickstart/spa/angular2/_includes/_centralized_login.md

```diff
@@ -87,7 +86,6 @@ export class AuthService {
   public handleAuthentication(): void {
     this.auth0.parseHash((err, authResult) => {
       if (authResult && authResult.accessToken && authResult.idToken) {
-        window.location.hash = '';
```

Comment:
Typically this is done to remove the authentication hash from the page history.  Is there a reason it is removed here?

Reply:
You're right, this line shouldn't be removed. When removing the authentication hash and using Firefox, the app get stuck in `http://localhost:3000/callback`. In Chrome and Opera the app works as expected.